templates/docker-compose/docker-compose.yml

```diff
@@ -111,13 +116,30 @@ services:
         networks:
             - supportpal_backend
 
+    meilisearch:
+        container_name: ${MEILISEARCH_SERVICE_NAME}
+        image: getmeili/meilisearch:v0.30.5
```

Comment:
They've released v1 5hrs ago, so we should look to use that

Reply:
> As a security measure, Meilisearch will now reject master keys that are less than 16 bytes in [production environment](https://docs.meilisearch.com/learn/configuration/instance_options.html#environment) ([#3274](https://github.com/meilisearch/meilisearch/pull/3274) and [#3295](https://github.com/meilisearch/meilisearch/pull/3295))

Seems to be the only breaking change that affects this PR.

https://github.com/meilisearch/meilisearch/releases/tag/v1.0.0